src/main/java/org/gooru/insights/services/ItemServiceImpl.java

```diff
@@ -507,7 +509,7 @@ public void run() {
 		} catch (AccessDeniedException accessDeniedException) {
```

Comment:
@daniel1028 , AccessDeniedException is throwing from validating user role function. And this exception is our custom exception so that only I have added throw here. Please confirm that its fine to remove.